cip/index.ts

```
@@ -0,0 +1,73 @@
+  newCommunityCreateEvent,
+  newCommunityInviteEvent,
+  newChannelCreateEvent,
+  newChannelMessageEvent,
+  toNostrEvent as toNostrEventCommunity
+} from './cip07/community.js'
+
+// Export core modules
+export * from './auth.js'
+export * from './constants.js'
+export * from './keys.js'
+export * from './subspace.js' 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)